plexus/templates/main/index.html

```diff
@@ -2,7 +2,7 @@
 
 {% block content %}
 
-<h1>Servers</h1>
+<a name="#servers"></a><h1>Servers</h1>
```

Comment:
It looks like the name attribute isn't supported in html5: http://www.w3schools.com/tags/att_a_name.asp
so this could be switched to `<a id="servers">`